### PR TITLE
[REF] stock_landed_costs_average: Separate landed cost lines creation

### PR DESCRIPTION
**Pull request by ebirbe**
We are separating creation of Landed Lost Line objects to be able to
adapt this object creation inside other modules.

Now if you want to change values of the landed cost line being created
you only need to overwrite this method and return the updated
dictionary.

This changes were proposed by @hbto to be used by other Vauxoo's projects.

### CODE REVIEW

File: stock_landed_costs_average/model/stock_landed_costs.py [hunk 1]
```diff
@@ -30,36 +30,36 @@ class StockLandedCost(models.Model):
...
-                    cost_lines.append((0, False, {
-                        'name': ail_brw.name,
-                        'account_id': ail_brw.account_id and
-                        ail_brw.account_id.id,
-                        'product_id': ail_brw.product_id and
-                        ail_brw.product_id.id,
-                        'price_unit': price_subtotal,
-                        'split_method': 'by_quantity',
-                    }))
+                    cost_lines.append((0, False, self.nuevo(ail_brw)))
```
Comment (hbto) [inline stock_landed_costs_average/model/stock_landed_costs.py]: @erickcion

Update method nuevo to the newly proposed one, 

And rebase against main branch it seems it is outdated. 

Regards

**Comment by ebirbe** [comment]
Hello @hbto. This is the PR we worked today.

Could you review it?

**Review by hbto — CHANGES_REQUESTED** [review]

File: stock_landed_costs_average/model/stock_landed_costs.py (hunk 1 above)
Comment (ebirbe) [reply stock_landed_costs_average/model/stock_landed_costs.py]: @hbto Done.

Rebased and fixed the method call.

**Comment by hbto** [comment]
+1

**Comment by JulioSerna** [comment]
👍

**Comment by JulioSerna** [comment]
[PR DUMMY](https://github.com/Vauxoo/typ/pull/69)

**Comment by JulioSerna** [comment]
we can fix lints in another pr

**Pull request merged**